tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/RcmBase.cs

```diff
@@ -23,4 +22,10 @@ public RcmBase(ITestOutputHelper outputHelper, string testName)
     {
         SetEnvironmentVariable(ConfigurationKeys.Rcm.PollInterval, "500");
     }
+
+    protected string AppSecDisabledMessage() => $"AppSec is now Disabled, _settings.Enabled is false, coming from remote config: true  {{ MachineName: \".\", Process: \"[{SampleProcessId}";
+
+    protected string AppSecEnabledMessage() => $"AppSec is now Enabled, _settings.Enabled is true, coming from remote config: true  {{ MachineName: \".\", Process: \"[{SampleProcessId}";
+
+    protected string RulesUpdatedMessage() => $"rules have been updated and waf status is \"DDWAF_OK\"  {{ MachineName: \".\", Process: \"[{SampleProcessId}";
```

Comment:
Is the pattern correct here? I think you'll need to use `*` instead of `.` as the wildcard?

Reply:
Seems the machine name in integration tests is always a dot , was mostly to avoid the use of a regex in the tests..

Reply:
oh... that's... surprising 😂 All good then!